### DISCUSSION TITLE
Alternative hardware

### DISCUSSION THREAD

**Discussion by BenGWeeks** [cryptoadvance/specter-diy#272] [General]
Have you consider building this for alternative hardware such as the Raspberry Pi with a large touchscreen?

**Comment by Garrysoon** [comment]
or Maix Amigo

**Comment by aurenlog** [comment]
Would you consider releasing a build/image for alternative hardware? The STM32F469I-DISCO is becoming extremely difficult to find, and at the moment it seems almost impossible to get one in stock anywhere in Europe. Supporting another readily available development board could make Specter-DIY much more accessible again.
